test/os-queens-environment.json

```diff
@@ -98,7 +98,7 @@
 		},
 		{
 			"key": "ADC_IMAGE",
-			"value": "13.1.1",
+			"value": "BIGIP-13.1.1",
```

Comment:
Can you please modify image name in another PR? This is not a part of your work, and we need this change right now.

Reply:
fixed